### PR TITLE
ba-109 Split up inventory-create and inventory-edit

### PR DESCRIPTION
**Pull request by lyio**
Create a new Polymer element socobo-inventory-edit
Moved edit item functionality over to the new element

### CODE REVIEW

File: app/elements/socobo-inventory/socobo-inventory.html [hunk 1]
```diff
@@ -130,8 +133,8 @@
         /* Display dialog to edit inventory items
         */
         _editItem: function(e) {
-          this.$.inventoryCreateDialog.style.display = "block";
-          this.$.inventoryCreateDialog.show(e.detail);
+          this.$.inventoryEditDialog.style.display = "block";
```
Comment (flashback2k14) [inline app/elements/socobo-inventory/socobo-inventory.html]: You doesn't need this. :host() set already display to block.

Comment (lyio) [inline app/elements/socobo-inventory/socobo-inventory.html]: Was wondering about that

**Comment by flashback2k14** [comment]
not working!
1. edit item -> click save button -> open create item dialog
2. measurement not showing correct in the list
   unbenannt
3. console errors:
   unbenannt_2

**Comment by flashback2k14** [comment]
i.O,

**Pull request merged**